css/styles.css

```diff
@@ -0,0 +1,255 @@
+
+/*black, white, 3 other colors rqd.*/
+
+/*Universal Styles*/
+* {
+	margin:0;
+	padding: 0;
+
```

Comment:
While there is no gospel truth when it comes to line-spacing in CSS, it is a good to be mindful of the way you are formatting. I noticed that the line spaces vary throughout your CSS, and it can affect readability for any teammates that you work with in the future. Not a huge deal, but definitely worth considering :) Let me know if you need me to clarify!